buildSrc/src/test/kotlin/com/datadog/gradle/plugin/jsonschema/TestDefinitions.kt

```diff
@@ -349,6 +372,63 @@ val Product = TypeDefinition.Class(
     )
 )
 
+val Bike = TypeDefinition.Class(
+    name = "Bike",
+    properties = listOf(
+        TypeProperty(
+            "productId",
+            TypeDefinition.Primitive(JsonPrimitiveType.INTEGER),
+            false,
+            defaultValue = 1.0
```

Comment:
nope...it's the way Gson interprets an `Any` that has value `1` . It will always parse it as a Double. Then we correct this in the PokoGenerator the same way you did for constants. I basically used the same method you were using for constants before, see the `getValueDefinitionForType` above.